README.md

```diff
@@ -264,6 +267,37 @@ Each organization/tenant is associated to a different database.
 
 [Top](#top)
 
+###<a name="section3.4"></a>OrionMongoSink persistence
```

Comment:
My fault, I forgot to comment that after this PR I will create a new one related to this issue https://github.com/telefonicaid/fiware-cygnus/issues/322. That will imply a full reorganization of the sink sections, most probably linking to the specific docs as proposed, and containing a minimal explanation on how the notification example within the README maps into real data structures at the backends.